src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int UPPER_EXCLUSIVE_BOUND_FOR_BALL_NUMBER = 101;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(),
+        random.nextInt(UPPER_EXCLUSIVE_BOUND_FOR_BALL_NUMBER));
```

Comment:
It would be great to place a `tab` here. In order to text on the 11 line is to the right of the text on the 12th.
```
return new Ball(colorSupplier.getRandomColor(),
        random.nextInt(UPPER_EXCLUSIVE_BOUND_FOR_BALL_NUMBER));
```